kodilla-cucumber/src/test/java/wallet/DisplayBalanceWalletSteps.java

```diff
@@ -0,0 +1,28 @@
+package wallet;
+
+import io.cucumber.java8.En;
+
+import static org.junit.Assert.assertEquals;
+
+public class DisplayBalanceWalletSteps implements En {
+
+    public DisplayBalanceWalletSteps() {
+
+        Wallet wallet = new Wallet();
+        CashSlot cashSlot = new CashSlot();
+        Teller teller = new Teller(cashSlot);
+
+        Given("my account has been credited with {int}", (Integer credit) -> {
+            wallet.deposit(credit);
+            teller.withdraw(wallet, credit);
+        });
+
+        Then("I should see that my balance is {int}", (Integer expectedBalance) -> {
+            assertEquals((int)expectedBalance, cashSlot.getContents());
+        });
+
+        When("I check my balance", () -> {
```

Comment:
czemu taka kolejność zapisu given - then -when?

Reply:
bo tak mi konsola wyrzucała, było given-when-then - lecz kazał mi usunąć when, więc usunęłam uzupełniłam metodami, to mi wtedy wyrzucił when i dodałam na końcu, bo tak chciał